wallet/src/de/schildbach/wallet/ui/TransactionsAdapter.java

```diff
@@ -247,16 +238,20 @@ public void onClick(final View v) {
                 }
             });
 
+            //
+            // TODO:  Fix merge conflicts with Transaction Details Screen
+            //
+
             if (onClickListener != null) {
-                transactionHolder.menuView.setOnClickListener(new View.OnClickListener() {
-                    @Override
-                    public void onClick(final View v) {
-                        onClickListener.onTransactionMenuClick(v, tx);
-                    }
-                });
+           //     transactionHolder.menuView.setOnClickListener(new View.OnClickListener() {
+           //         @Override
+           //         public void onClick(final View v) {
+           //             onClickListener.onTransactionMenuClick(v, tx);
+           //         }
+           //     });
```

Comment:
these comments are left in for now.  This section of code will probably have merge conflicts with #263.